README.md

```diff
@@ -23,6 +23,20 @@ git add -A
 git commit -m "Your commit message"
 git push --set-upstream origin new-branch
 ```
+   The git add command lets you add a file to the staging area. You can think of the staging area as a step before the commit process      in git.
+   As long as a changeset is in the staging area, git allows you to edit it as you like.
+   The git add -A command however, finds all new and updated files everywhere throughout the project and add them to the staging area. 
+
+* Inspecting a repository
```

Comment:
Take a look at **github markdown** to format the text in .md files [link](https://guides.github.com/features/mastering-markdown/)

Reply:
You can format your changes using markdown :) for better readibility